mohammad/CSS/CSS-flexbox/style.css

```diff
@@ -0,0 +1,100 @@
+.row1 {
+  width: 300px;
+  height: 150px;
+  display: flex;
+  flex-direction: row;
+}
+.r1c1 {
```

Comment:
This file is unnecessary. You have 2 CSS files in this PR, we only need one. Make sure you keep the right file, Also try to not to close this PR and commit the needed changes needed the same way you did the last 4 commits.